UM/Qt/Bindings/Theme.py

```diff
@@ -135,6 +137,14 @@ def getIcon(self, icon_name: str, detail_level: str = "default") -> QUrl:
         elif icon_name in self._icons["icons"]:  # Retrieve the "old" icon from the base icon folder
             return self._icons["icons"][icon_name]
 
+        if icon_name in self._deprecated_icons:
+            new_icon = self._deprecated_icons[icon_name]["new_icon"]
+            warning = "The icon {old_icon_name} is deprecated. Please use {new_icon} instead.".format(old_icon_name = icon_name, new_icon = new_icon)
```

Comment:
We can actually use modern string formatting nowadays, new in Python 3.6!

```suggestion
            warning = f"The icon {icon_name} is deprecated. Please use {new_icon} instead."
```

Reply:
I keep forgetting this. Good call!